core/src/main/java/org/apache/calcite/rel/rel2sql/RelToSqlConverter.java

```diff
@@ -633,6 +634,21 @@ public Result visit(Project e) {
       if ((!isStar(e.getProjects(), e.getInput().getRowType(), e.getRowType())
           || style.isExpandProjection()) && !unpivotRelToSqlUtil.isStarInUnPivot(e, x)) {
         final List<SqlNode> selectList = new ArrayList<>();
+
+        List<String> aliases = new ArrayList<>();
```

Comment:
can be named as` pivotAliases/pivotColumnAliases`, if thats what it denotes

Reply:
Renamed